site/en/blog/new-in-chrome-94/index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: New in Chrome 94
+description: >
+  Chrome 94 is rolling out now! The default color space for `<canvas>`
+  elements is now formally defined as sRBG, and you can change it to Display
+  P3. There's a new, low level way to access built in audio and video codecs,
+  important for streaming games, video editors, and such. WebGPU starts its
+  origin trial. And, there's plenty more!
+layout: 'layouts/blog-post.njk'
+date: 2021-09-21
+authors:
+  - petelepage
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/bFEcnAWdzNkXaSSp0hX4.png'
+alt: >
+  New in Chrome hero logo
+tags:
+  - new-in-chrome
+  - chrome-94
+---
+
+{% YouTube id='c5XIDt90VgY' %}
+
+Here's what you need to know:
+
+* The [default color space](#canvas-colorspace) for `<canvas>` elements is
+  now formally defined  in the spec as *sRBG*, and you can change it to
+  *Display P3*.
+* [WebCodecs](#webcodecs) is a new, low level way to access built in audio and
+  video codecs, important for streaming games, video editors, and such.
+* [WebGPU](#webgpu) starts its origin trial.
+* The [PWA Summit](#pwa-summit) is coming up October 6-7.
+* And there's plenty [more](#more).
+
+I'm [Pete LePage](https://petelepage.com), working, and shooting
+from home, let's dive in and see what's new for developers in Chrome 94.
+
+## Default color space for `canvas` elements {: #canvas-colorspace }
+
+How color is rendered on screen is critical to some users. For photographers,
+print illustrators, and many others, the colors on screen need to match what's
+printed. Starting in [Chrome 94][cs-canvas], `<canvas>` elements are fully
+color managed using *sRGB*. Previously, *sRGB* was convention, but not
+explicitly defined in the spec.
+
+```js
+opts = {colorSpace:'display-p3'};
+const ctx = canvas.getContext('2d', opts);
+```
+
+More importantly - you can now specify which color space to use when creating
+a `<canvas>` rendering context 2d object, or an `ImageData` object, including
+the P3 color space.
+
+## WebCodecs {: #webcodecs }
+
+Putting video on a page is easy enough. But, if you need to do something a
+little more complex, and interact with the components of a video stream,
+it's hard! And, typically requires you to use Web Assembly to ship
```

Comment:
```suggestion
it's hard, and typically requires you to use Web Assembly to ship
```

Reply:
The other option was to add a verb to the second sentence ("And *it* typically...). Since this is a transcript...